src/VirtualMonoRepo/source-mappings.json

```diff
@@ -64,7 +64,8 @@
             "name": "aspire",
             "defaultRemote": "https://github.com/dotnet/aspire",
             "exclude": [
-                "src/Aspire.Dashboard/**/*"
+                "src/Aspire.Dashboard/**/*",
+                "playground/eShopLite/**/*"
```

Comment:
It looks like the entire `playground` directory could be excluded here. It's not used in the build.